sources/packages/backend/workflow/BPMN/camunda-8/parttime-configure-disbursement.bpmn

```diff
@@ -52,7 +52,7 @@
     <bpmn:endEvent id="Event_0ttwbth">
       <bpmn:incoming>Flow_1p03d1g</bpmn:incoming>
     </bpmn:endEvent>
-    <bpmn:intermediateThrowEvent id="Event_1koar33" name="Configure Payload">
+    <bpmn:intermediateThrowEvent id="Event_1koar33" name="Configure Payload for 17 weeks">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=[&#10;  {&#10;    disbursementDate: disbursmenentScheduleDate1,&#10;    negotiatedExpiryDate: disbursmenentScheduleDate1,&#10;    disbursements: [&#10;      {&#10;        valueCode: &#34;CSLP&#34;,&#10;        valueType: &#34;Canada Loan&#34;,&#10;        valueAmount: decimal(finalFederalAwardNetCSLPAmount,0),&#10;        awardEligibility: true&#10;      },&#10;      {&#10;        valueCode: &#34;CSGP&#34;,&#10;        valueType: &#34;Canada Grant&#34;,&#10;        valueAmount: decimal(finalFederalAwardNetCSGPAmount, 0),&#10;        awardEligibility: awardEligibilityCSGP&#10;      },&#10;      {&#10;        valueCode: &#34;CSGD&#34;,&#10;        valueType: &#34;Canada Grant&#34;,&#10;        valueAmount: decimal(finalFederalAwardNetCSGDAmount, 0),&#10;        awardEligibility: awardEligibilityCSGD&#10;      },&#10;      {&#10;        valueCode: &#34;CSPT&#34;,&#10;        valueType: &#34;Canada Grant&#34;,&#10;        valueAmount: decimal(finalFederalAwardNetCSPTAmount, 0),&#10;        awardEligibility: awardEligibilityCSPT&#10;      },&#10;      {&#10;        valueCode: &#34;BCAG&#34;,&#10;        valueType: &#34;BC Grant&#34;,&#10;        valueAmount: decimal(finalProvincialAwardNetBCAGAmount, 0),&#10;        awardEligibility: awardEligibilityBCAG&#10;      },&#10;      {&#10;        valueCode: &#34;SBSD&#34;,&#10;        valueType: &#34;BC Grant&#34;,&#10;        valueAmount: decimal(finalProvincialAwardNetSBSDAmount, 0),&#10;        awardEligibility: awardEligibilitySBSD&#10;      }&#10;    ]&#10;    [item.awardEligibility != false]&#10;  }&#10;]" target="disbursementSchedules" />
```

Comment:
Probably not PR-related but this is the only place saving the awards using the `decimal` method to do the rounding instead of the `ceiling` method and I have no idea why.
Does anyone remember?
@dheepak-aot @guru-aot @sh16011993

Reply:
I don't remember the reason.

Reply:
I do not remember.

Reply:
I replaced decimal with ceiling.